### DISCUSSION TITLE
Missing icons

### DISCUSSION THREAD

**Discussion by luispuerto** [elrumo/macOS_Big_Sur_icons_replacements#1164] [Help with the website] [answered]
Hey! 

Seems that you still have some missing icons since de security problem. I was trying to reupload the ones I have around on png (I have most of them on icns) but it's really tedious to put all the information on a butch of icons. Is there any other way to share with you? 

Thanks!

**Comment by rodgerwerner** [comment, marked as answer]
As far as I am aware, @Elias is working on the new system, allowing users to have accounts.  I would assume that once this has been updated/activated, users will be able to re-upload or edit existing uploads etc. All it takes from us is a little patience ... eventually things will be back to normal.

**Reply by luispuerto** [reply]
I've asked because I think that at some point when I uploaded at icon, it was a message on the site asking if we have previous uploaded icons to upload them. 

Also yesterday someone wrote me an email asking for one of my uploaded icons since it wasn't available on the site.
